src/components/app-select/AppSelect.jsx

```diff
@@ -0,0 +1,39 @@
+import { useTranslation } from 'react-i18next'
+
+import MenuItem from '@mui/material/MenuItem'
+import Select from '@mui/material/Select'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+
+import { styles } from '~/components/app-select/AppSelect.styles'
+
+const AppSelect = ({ setSortBy, sortBy, sortingFields, selectTitle = '' }) => {
+  const { t } = useTranslation()
+
+  const changeSortBy = (e) => setSortBy(e.target.value)
+
+  const sortingFieldsList = sortingFields.map(field => (
+    <MenuItem key={ field.value } value={ field.value }>
+      { t(field.title) }
+    </MenuItem>)
+  )
+
+  return (
+    <Box sx={ styles.selectContainer } >
+      <Typography sx={ styles.selectTitle } variant='subtitle1' >
+        { t(selectTitle) }
+        :
+      </Typography>
+      <Select
+        inputProps={ { 'data-testid': 'sort-by-select' } }
+        onChange={ changeSortBy }
+        sx={ styles.selectField }
+        value={ sortBy }
```

Comment:
To my mind sortBy is not right name, it`s more like sortedValue or smth like that